iXBRLViewerPlugin/LocalViewer.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2019 Mark V Systems Limited, except as noted below.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Portions are from Edgar(tm) Renderer which was created by staff of the U.S. Securities and Exchange Commission.
+# Data and content created by government employees within the scope of their employment 
+# are not subject to domestic copyright protection. 17 U.S.C. 105.
```

Comment:
Im not real sure how all this copyright stuff works. It seems a little weird that this file alone would be copyrighted by `Mark V Systems` and the rest of the repo by `Workiva`. Not suggesting we change it, but just calling it out as an inconsistency in the copyrighting.

Reply:
the Mark V stuff was refactored to infrastructure, now this module uses W's legalese